android/src/main/java/com/mr/flutter/plugin/filepicker/FilePickerDelegate.java

```diff
@@ -208,6 +209,15 @@ private static void finishWithAlreadyActiveError(final MethodChannel.Result resu
         result.error("already_active", "File picker is already active", null);
     }
 
+    @SuppressWarnings("deprecation")
+    private ArrayList<Parcelable> getSelectedItems(Bundle bundle){
+        if(Build.VERSION.SDK_INT >= 33){
```

Comment:
I saw that there was a similar issue with using the Tiramisu constant. Shouldn't we encourage people to update to the latest compile SDK version? (and fix the two usages of `33` in the codebase)